### DISCUSSION TITLE
Global/local chat spy

### DISCUSSION THREAD

**Discussion by div2005** [Zrips/CMI#3260] [Enhancement]
There is a 'cmi.chat.rangebypass' permission that allows people with it to see and send messages from any distance on the server. However, this can become inconvenient to use for supervision of local chat to prevent abuse and server rule violation. This is because it removes the ability to write in local chat. Chat differentiation is very useful if server moderators/administrators need to convey some information for nearby players.
Summary: Add a permission or command that allows spying on local chat across the server  without messing with ability to sent local and global messages separately.

**Comment by div2005** [comment]
Up

**Comment by div2005** [comment]
up

**Comment by RemForever0** [comment]
up

**Comment by CloudeLecaw** [comment]
upping doesnt help. all discussions are seen or not. the place in list doesnt matter.
